src/tests/main.test.ts

```diff
@@ -458,3 +458,10 @@ describe("Purchases.close()", () => {
     expect(Purchases.isConfigured()).toBeFalsy();
   });
 });
+
+describe("Purchases.initialize()", () => {
+  test("can initialize without failing", async () => {
+    const purchases = configurePurchases();
+    await purchases.initialize();
```

Comment:
I would like to write better tests for this, but currently the side effects of this operation stay inside the `purchases` object, and are only noticeable in the `purchase` method, which is not very testable through unit tests since it mostly displays a UI...